cpp/open3d/core/Tensor.cpp

```diff
@@ -750,7 +750,7 @@ std::string Tensor::ToString(bool with_suffix,
     std::ostringstream rc;
     if (IsCUDA() || !IsContiguous()) {
         Tensor host_contiguous_tensor = Contiguous().To(Device("CPU:0"));
-        rc << host_contiguous_tensor.ToString(false, "");
+        rc << host_contiguous_tensor.ToString(with_suffix, indent);
```

Comment:
Sorry, I added a regression here. `with_suffix` should be set to false, else it will print suffix twice. once in inner call where tensor is moved to CPU and once in outer call.

Example - 
```
o3d.core.Tensor.eye(n=4, dtype=o3d.core.Dtype.Float64, device=o3d.core.Device('cuda:0'))
[[1 0 0 0],
 [0 1 0 0],
 [0 0 1 0],
 [0 0 0 1]]
Tensor[shape={4, 4}, stride={4, 1}, Float64, CPU:0, 0x561c38e195d0]
Tensor[shape={4, 4}, stride={4, 1}, Float64, CUDA:0, 0x302000000]
```

I will address this in future PR.